### DISCUSSION TITLE
What would cause message “ Failed to get any guids for”

### DISCUSSION THREAD

**Discussion by Splinter7914** [luigi311/JellyPlex-Watched#148] [Q&A]
Getting started with setting up this tool and in the logs i see the message `Failed to get any guids for` show up quite a bit. 

I was curious what could cause this?I have debug logs enabled and there are no other errors around it. Is this expected or does this mean something is off with my setup?

**Comment by Splinter7914** [comment]
oops i had debug true but level still set to info :).

**Reply by popnfresh11** [reply]
Did you solve why this was happening? I'm also getting this error but not sure why and not well versed in debugging either...
